packages/consumption/src/modules/requests/incoming/IncomingRequestsController.ts

```diff
@@ -174,10 +174,10 @@ export class IncomingRequestsController extends ConsumptionBaseController {
         const relationship = await this.relationshipResolver.getRelationshipToIdentity(request.peer);
         // It is safe to decide an incoming Request when no Relationship is found as this is the case when the Request origins from onNewRelationship of the RelationshipTemplateContent
         if (relationship) {
-            if (typeof relationship.peerDeletionInfo?.deletionStatus !== "undefined") {
+            if (relationship.peerDeletionInfo?.deletionStatus) {
```

Comment:
I mean that you don't need two `if`s here, they can be combined into one `relationship && relationship.peerDeletionInfo?.deletionStatus` and this can be further abbreviated into `relationship?.peerDeletionInfo?.deletionStatus`